pkg/console/starter/starter.go

```diff
@@ -456,20 +456,29 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	)
 
 	// Show all ConsolePlugin instances as related objects
+	// Show all namespaces that are used by ConsolePlugin instances as related objects
 	clusterOperatorStatus.WithRelatedObjectsFunc(func() (bool, []configv1.ObjectReference) {
 		relatedObjects := []configv1.ObjectReference{}
 		consolePlugins, err := consoleClient.ConsoleV1().ConsolePlugins().List(ctx, metav1.ListOptions{})
 		if err != nil {
 			return false, nil
 		}
 		for _, plugin := range consolePlugins.Items {
-			relatadPlugin := configv1.ObjectReference{
+			relatedObjects = append(relatedObjects, configv1.ObjectReference{
 				Group:    "console.openshift.io",
 				Resource: "consoleplugins",
 				Name:     plugin.GetName(),
+			})
+			if plugin.Spec.Backend.Service != nil {
```

Comment:
We might include any plugin proxy namespaces as well.

Should we de-dup the list if multiple plugins are served from the same namespace? (Or if the plugin namespace is the same as one of the hard-coded ones, openshift-console or openshift-console-operator?)

Reply:
Added
- namespaces from the ConsolePluginProxy 
- deduplication of the relatedObjects
- unitTest

Thank you @spadgett PTAL